docs/howtos/DataPlane Generation - DPG/08methodInputs.mdx

```diff
@@ -0,0 +1,456 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Method Inputs
+
+This doc details what method input signatures language emitters will output.
+
+## Basic
+
+### HTTP Get
+
+<Tabs>
+<TabItem value="typespec" label="TypeSpec" default>
+
+```typespec
+op @get get(): User;
+```
+
+</TabItem>
+
+<TabItem value="python" label="Python">
+
+```python
+def get() -> User:
+  ...
+```
+
+</TabItem>
+<TabItem value="csharp" label="CSharp" >
+
+```csharp
+```
+
+</TabItem>
+<TabItem value="typescript" label="Typescript" >
+
+```typescript
+```
+
+</TabItem>
+<TabItem value="java" label="Java" >
+
+```java
+```
+
+</TabItem>
+<TabItem value="go" label="Go" >
+
+```go
+```
+
+</TabItem>
+</Tabs>
+
+### HTTP Post
+
+<Tabs>
+<TabItem value="typespec" label="TypeSpec" default>
+
+```typespec
+op @post post(User): void;
```

Comment:
```suggestion
@post op post(@body user: User): void;
```

Reply:
with/without @body decorator should have different behavior.  Does this part want to show the scenaro without @body decorator which mean anonymous body?

Reply:
I just mean `post(User)` is not valid in typespec... I just guess Isabella's intended code, not sure.

Reply:
this case aims to show the basic usage, so weidong is right, i changed the doc.